lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -669,6 +669,80 @@ describe('modules/manager/npm/extract/index', () => {
         ],
       });
     });
+
+    it('extracts dependencies from overrides', async () => {
+      const content = `{
+        "devDependencies": {
+          "@types/react": "18.0.5"
+        },
+        "overrides": {
+          "node": "8.9.2",
+          "@types/react": "18.0.5",
+          "baz": {
+            "bar": {
+              "foo": "1.0.0"
+            }
+          },
+          "foo2": {
+            ".": "1.0.0",
+            "bar2": "1.0.0"
+          },
+          "emptyObject":{}
+        }
+      }`;
+      const res = await npmExtract.extractPackageFile(
+        content,
+        'package.json',
+        defaultConfig
+      );
+      expect(res).toMatchSnapshot({
```

Comment:
```suggestion
      expect(res).toMatchObject({
```
i don't think we need the snapshot